cxmon/src/mon.cpp

```diff
@@ -153,7 +154,7 @@ void mon_add_command(const char *name, void (*func)(), const char *help_text)
 
 void mon_error(const char *s)
 {
-	fprintf(monerr, "*** %s\n", s);
+	fprintf(monerr == NULL? stdout: monerr, "*** %s\n", s);
```

Comment:
Nit: Add a space after stdout, otherwise it looks confusing.

Reply:
Done